packages/syft/src/syft/util.py

```diff
@@ -55,6 +53,10 @@ def validate_type(_object: object, _type: type, optional: bool = False) -> Any:
     )
 
 
+def get_syft_built() -> ModuleType:
+    return sys.modules[__name__.split(".")[:-1][0]]
```

Comment:
Hmm why not just `__name__.split(".")[0]`? The only case when there's a difference is when `__name__.split(".")` has length 1 but then it would throw an error?

Reply:
Q: Why did we replace from "syft" string to using `__name__`?

Reply:
also NIT but should it be `get_syft_build()` instead of `built`?

Reply:
> Hmm why not just `__name__.split(".")[0]`? The only case when there's a difference is when `__name__.split(".")` has length 1 but then it would throw an error.

Yup, you're right, not only that but the current solution isn't reliable if we change this source file to another place. Thanks for noticing it.

Reply:
> Q: Why did we replace from "syft" string to using `__name__`?

Not a cybersecurity expert here. 
But I thought in a scenario where `syft` key is overwritten in `sys.modules` making this package load a malicious package inside of it. Not completely sure if it's feasible but better safe than sorry

Reply:
> also NIT but should it be `get_syft_build()` instead of `built`?

The main purpose of this method is to get syft package when its completely loaded, that's why I named it with "built". but now, thinking a bit about that, I guess a better name would be `get_loaded_syft`.

Reply:
Solved by: [5194633](https://github.com/OpenMined/PySyft/pull/7030/commits/51946335e520361eeda3824d64a3796e77e9c511) and [91c73f1](https://github.com/OpenMined/PySyft/pull/7030/commits/91c73f19ebf8a7d552168b69c6b56fc3ad5a6d7d)

Reply:
> > Q: Why did we replace from "syft" string to using `__name__`?
> 
> I'm not a cybersecurity expert. But I thought in a scenario where `syft` key is overwritten in `sys.modules` making this package load a malicious package inside of it. Not completely sure if it's feasible but better safe than sorry

I see, good point 👍